.gitignore

```diff
@@ -2,6 +2,8 @@
 .tox
 .cache
 .pytest_cache/
+.idea/
+.python-version
```

Comment:
IMO these are down to individual preferences so not part of the project.
I have mine in ~/.git_ignore_global

Reply:
Thx, I did not planned to add this commit though, i just wanted to launch a build on Travis-CI (as it seems there are no way to trigger a build manually bu on master branch).
Thanks for the tip on `~/.git_ignore_global` :)